src/Place/Canonical/ClusterRecordRow.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Place\Canonical;
+
+class ClusterRecordRow
```

Comment:
Maybe this is a `PlaceWithClusterRecord`? Because a row is always unique on place id.